src/ipc/mount-usb-drive.ts

```diff
@@ -9,13 +9,18 @@ export interface Options {
   label?: string;
 }
 
+const MOUNT_DIRECTORY = '/media/vx/usb-drive';
+const MOUNT_OPTIONS = ['umask=000', 'nosuid', 'nodev', 'noexec'];
```

Comment:
These options are protections mimicking the same way we've been mounting with `pmount`.